src/main/groovy/org/boozallen/plugins/jte/binding/LibraryLoader.groovy

```diff
@@ -30,14 +30,30 @@ import org.jenkinsci.plugins.workflow.cps.CpsScript
         // 1. Inject steps from loaded libraries
         List<GovernanceTier> tiers = GovernanceTier.getHierarchy() 
         List<TemplateLibrarySource> sources = tiers.collect{ it.librarySources }.flatten().minus(null)
-        
+
+        ArrayList libConfigErrors = []
         config.getConfig().libraries.each{ libName, libConfig ->
             TemplateLibrarySource s = sources.find{ it.hasLibrary(libName) }
             if (!s){ 
-                throw new TemplateConfigException("Library ${libName} Not Found.") 
+                libConfigErrors << "Library ${libName} Not Found." 
+            } else {
```

Comment:
changed 
if (!s){ 
                throw new TemplateConfigException("Library ${libName} Not Found.") 	                libConfigErrors << "Library ${libName} Not Found." 
            } 
            libConfigErrors << s.loadLibrary(script, libName, libConfig)
---
to
if (!s){ 
                throw new TemplateConfigException("Library ${libName} Not Found.") 	                libConfigErrors << "Library ${libName} Not Found." 
            } else {
                libConfigErrors << s.loadLibrary(script, libName, libConfig)
            }
---
assuming the former would generate a null pointer